code/modules/client/preferences/species_features/basic.dm

```diff
@@ -122,7 +122,7 @@
 	savefile_key = "hair_color"
 	savefile_identifier = PREFERENCE_CHARACTER
 	category = PREFERENCE_CATEGORY_SUPPLEMENTAL_FEATURES
-	relevant_species_trait = HAIR
+	//relevant_species_trait = HAIR
```

Comment:
Why was commenting this out required?

Reply:
As best as I can understand it, commenting this out is required to change hair_color on a species that doesn't _technically_ have hair (podperson "hair" is an external organ). Without this, attempting to add a hair color selector to the preferences menu crashes the menu.

Reply:
I see.